### DISCUSSION TITLE
According to legend, how did Alexander the Great untie the Gordian Knot?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#315] [Galaxy Trivia] [answered]
**Hint:** ⚔️

**Comment by thuongtruong1009** [comment, marked as answer]
According to legend, Alexander the Great untied the Gordian Knot by slicing it in half with his sword. The Gordian Knot was a complex knot tied by King Gordius of Phrygia, and an oracle had declared that whoever could untie the knot would become the ruler of all of Asia. Alexander, who was still a young man at the time and not yet the ruler of a vast empire, was visiting the city of Gordium when he encountered the knot. Unable to find the end of the knot to untie it, he simply took out his sword and cut it in half, thus fulfilling the prophecy in his own way.

**Reply by ParanoidUser** [reply]
> According to legend, Alexander the Great untied the Gordian Knot by slicing it in half with his sword. The Gordian Knot was a complex knot tied by King Gordius of Phrygia, and an oracle had declared that whoever could untie the knot would become the ruler of all of Asia. Alexander, who was still a young man at the time and not yet the ruler of a vast empire, was visiting the city of Gordium when he encountered the knot. Unable to find the end of the knot to untie it, he simply took out his sword and cut it in half, thus fulfilling the prophecy in his own way.

[International Guild of Knot Tyers](https://igkt.net) may need to consider banning such untying techniques. 😄 Well done!
